src/chainstate/burn/db/sortdb.rs

```diff
@@ -6807,16 +6822,19 @@ pub mod tests {
                     epoch_id: StacksEpochId::Epoch10,
                     start_height: 0,
                     end_height: 8,
+                    block_limit: ExecutionCost::max_value(),
                 },
                 StacksEpoch {
                     epoch_id: StacksEpochId::Epoch20,
                     start_height: 8,
                     end_height: 12,
+                    block_limit: ExecutionCost::max_value(),
```

Comment:
Shouldn't these use the proper max values for their respective epochs?

Reply:
This test isn't doing any checks of the `block_limit`, it's just testing the sortition db's epoch tracking functionality, so using `max_value()` (or really any value) doesn't matter in this test.